data/eos-autoupdater.conf

```diff
@@ -10,4 +10,5 @@
 #  3 - apply updates
 LastAutomaticStep=3
 IntervalDays=14
+RandomizedDelayDays=7
```

Comment:
I just don't want to introduce any policy changes for everyone as part of this. If you think there's merit in a different approach for regular users, please open a separate story.